libs/math/tests/unit/basic_math/big_number_tests.cpp

```diff
@@ -177,26 +177,24 @@ TEST(big_number_gtest, to_double_tests)
 }
 
 template <typename LongUInt>
-bool IsTrimmedSizeValid()
+void TestTrimmedWideSize()
 {
-  bool     isValid = LongUInt(uint64_t(0)).TrimmedSize() == 1;
+  EXPECT_EQ(0, LongUInt(uint64_t(0)).TrimmedWideSize());
```

Comment:
I guess TrimmedSize() of a zero is still 1, because "0" is still a valid number, just occupying the least wide element?

Reply:
The Trimmed*Size() methods follow the size idiom used in containers (e.g. native C++ array, or std containers like std::array<...>, std::list<...>, etc. ...), so that methods can be used in cycles - for example `for (std::size_t i =0; i<TrimmedSize(); ++i) {...}` where cycle won't execute if `TrimmedSize()` returns `0`.